src/storage/src/sst.rs

```diff
@@ -121,23 +136,27 @@ pub struct LevelMeta {
     /// Handles to the files in this level.
     // TODO(yingwen): Now for simplicity, files are unordered, maybe sort the files by time range
     // or use another structure to hold them.
-    files: Vec<FileHandle>,
+    files: HashMap<String, FileHandle>,
```

Comment:
Using string here is a bit wasteful because we use uuid (128bit) as file name. Could we just use define a struct for the file name and only generate the string when we are going to read the file from the object store?